src/components/MultiHopTrade/components/MultiHopTradeConfirm/components/ApprovalStep.tsx

```diff
@@ -124,7 +126,7 @@ const ApprovalStepPending = ({
   const checkLedgerAppOpenIfLedgerConnected = useLedgerOpenApp()
 
   const { state, allowanceReset, approval } = useAppSelector(state =>
-    selectHopExecutionMetadata(state, hopIndex),
+    selectHopExecutionMetadata(state, tradeId, hopIndex),
```

Comment:
Same q. here - should `<ApprovalStep />` `selectActiveQuoteId`, or do we want to keep passing it from up-top in `<Hops />`?